### PR TITLE
chore: week39 taxonomy updates

### PR DESCRIPTION
**Pull request by aleene**
### What
Request etc.

single ingredient / price / quality monitoring additions

### CODE REVIEW

File: taxonomies/food/ingredients.txt [hunk 1]
```diff
@@ -52397,20 +52402,42 @@ wikidata:en: Q35075688
...
+fr: asperges blanches, asperge blanche, asperges blanches grosses, asperges blanches des sables des Landes IGP
 hr: bijele šparoge
 it: asparagi bianchi
 nl: witte asperges
 pl: białe szparagi, biały szparag, białych szparagów
 pt: Espargos brancos
 # fr:asperges blanches has 46 products in 4 languages @2018-11-11
 
+< en:white asparagus
+fr: asperges blanches coupées
```
Comment (stephanegigandet) [inline taxonomies/food/ingredients.txt:52424]: we should not need this one, "coupées" is in processing

File: taxonomies/food/ingredients.txt [hunk 2]
```diff
@@ -52376,6 +52378,9 @@ eurocode_2_group_3:en: 8.25.10
 wikidata:en: Q23041045
 wikipedia:en: https://en.wikipedia.org/wiki/Asparagus
 
+< en:asparagus
+fr: asperges fraîches
```
Comment (stephanegigandet) [inline taxonomies/food/ingredients.txt:52382]:
```suggestion
en: fresh asparagus
fr: asperges fraîches
```

File: taxonomies/food/ingredients.txt (hunk 1 above)
Comment (aleene) [reply taxonomies/food/ingredients.txt:52424]: agree, reverted

**Review by stephanegigandet — APPROVED** [review]
Thanks!

**Pull request merged**